superset/models/dashboard.py

```diff
@@ -156,7 +156,7 @@ class Dashboard(  # pylint: disable=too-many-instance-attributes
     ]
 
     def __repr__(self) -> str:
-        return f"Dashboard<{self.slug or self.id}>"
+        return f"Dashboard<{self.id or self.slug}>"
```

Comment:
shouldn't this be the other way round? Is this one of those lines you didn't touch?

Reply:
I guess if it's for the `__repr__` function it makes more sense to use the id

Reply:
I haven't changed these lines, I think this is GH acting up showing some changes that have come in after the branch was cut.

Reply:
Huh. Maybe you merged master or something